### ISSUE TITLE
Finding latest NFive version... error

### ISSUE THREAD

**Issue by Antonizz** [NFive/nfpm#38]
I run the setup command in nfpm.exe and i setted all up but when i pressed yes to install nfive it says "An unhandled application error has occurred: Not Found".
![image](https://user-images.githubusercontent.com/69055232/127780070-edb11f6e-30ba-4cb6-95ca-e6defdbc34bc.png)

**Comment by Simmesh** [comment]
I have the same issue. @Antonizz Did you make it in vs somehow do not get it done

**Comment by amir-halloul** [comment]
It's working fine when I try it. Are you sure you are using the right version? Use `nfpm version` and make sure it's `NFive Plugin Manager 0.4 Build 236`

**Comment by Simmesh** [comment]
![image](https://user-images.githubusercontent.com/76971366/128070597-ea8b4d3a-5be2-481b-adb1-f04d941599ff.png)
yes is use this version

**Comment by Simmesh** [comment]
And when i go on:  https://dl.nfive.io/nfive.zip     or on:   https://dl.nfive.io/nfpm.exe
i get this error
![image](https://user-images.githubusercontent.com/76971366/128070830-4101c722-469d-4e4b-a08c-7231bd04c69c.png)

**Comment by Simmesh** [comment]
I found out that a "NotFoundException" occurs in this line 292 at InstallNFive in Setup.cs
![image](https://user-images.githubusercontent.com/76971366/128087589-b772c23a-b794-41e5-bd06-2be06e4d8710.png)
I think the reason for this error is that no released version is in NFive

**Comment by amir-halloul** [comment]
> 
> 
> yes is use this version

Yeah my bad. I thought the error happened while finding the latest FiveM version not NFive. Like @Simmesh pointed out, nfpm tries to find the latest NFive release from the github repo but we don't have any releases yet.
You can compile NFive yourself and add it to your server manually for now until we add a release for NFive.
